M2/Macaulay2/packages/SCMAlgebras.m2

```diff
@@ -0,0 +1,394 @@
+-- -*- coding: utf-8 -*-
+
+newPackage(
+    "SCMAlgebras",
+    Version => "1.0",
+    Date => "October 2, 2024",
+    Authors => {{Name => "Ernesto Lax", Email => "erlax@unime.it", HomePage => "https://www.researchgate.net/profile/Ernesto-Lax"}},
+    Headline => "a package to work with sequentially Cohen-Macaulay modules or ideals",
```

Comment:
The tests are currently failing because this headline includes the redundant "a package to ...".  From the [Package writing style guide](https://github.com/Macaulay2/M2/wiki/Package-Writing-Style-Guide):

> The Headline option to "newPackage" should be a brief phrase stating the topic of the package. For example, the headline for the package "Tropical" is the phrase "tropical geometry". It would be redundant to say something such as "a package for tropical geometry", "computations in tropical geometry", or "a package for computations in tropical geometry".